### PR TITLE
Prep 1.0.0 changelog

### PR DESCRIPTION
**Pull request by sverhoeven**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,11 +5,46 @@ Change Log
...
+`1.0.0`_ (2021-07-22)
+*********************
 
 Added
 -----
 
+* Documentation
+  * Example notebooks
+  * Setup guide (`#120 <https://github.com/eWaterCycle/ewatercycle/issues/120>`_)
+  * HPC cluster guide
```
Comment (SarahAlidoost) [inline CHANGELOG.rst:21]: The second level bullet items don't render correctly, see https://github.com/eWaterCycle/ewatercycle/blob/changelog-1.0.0/CHANGELOG.rst. I think we should use a different syntax like `-`.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,11 +5,46 @@ Change Log
...
+* Forcing generation using `ESMValTool <https://www.esmvaltool.org/>`_ (`#28 <https://github.com/eWaterCycle/ewatercycle/issues/28>`_, `#87 <https://github.com/eWaterCycle/ewatercycle/issues/87>`_,)
+* Available parameter sets (`#118 <https://github.com/eWaterCycle/ewatercycle/issues/110>`_)
+* `PyMT <https://pymt.readthedocs.io/>`_ inspired interface for following models
+  * LISFLOOD
+  * MARRMoT M01 and M14
+  * PCR-GLOBWB
+  * wflow
```
Comment (SarahAlidoost) [inline CHANGELOG.rst:30]: the same problem with bullet items.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,11 +5,46 @@ Change Log
...
 
 Added
 -----
 
+* Documentation
+  * Example notebooks
+  * Setup guide (`#120 <https://github.com/eWaterCycle/ewatercycle/issues/120>`_)
+  * HPC cluster guide
+* Forcing generation using `ESMValTool <https://www.esmvaltool.org/>`_ (`#28 <https://github.com/eWaterCycle/ewatercycle/issues/28>`_, `#87 <https://github.com/eWaterCycle/ewatercycle/issues/87>`_,)
+* Available parameter sets (`#118 <https://github.com/eWaterCycle/ewatercycle/issues/110>`_)
```
Comment (SarahAlidoost) [inline CHANGELOG.rst]:
```suggestion
* Available parameter sets (`#118 <https://github.com/eWaterCycle/ewatercycle/issues/118>`_)
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,11 +5,46 @@ Change Log
...
+  * LISFLOOD
+  * MARRMoT M01 and M14
+  * PCR-GLOBWB
+  * wflow
+* Containerized models using `grpc4bmi <https://github.com/eWaterCycle/grpc4bmi>`_
+* Configuration files for system setup
+* Hydrograph plotting (`#54 <https://github.com/eWaterCycle/ewatercycle/issues/54>`_)
+* Typings
+* iso8601 time format (`#90 <https://github.com/eWaterCycle/ewatercycle/issues/90>`_)
+
```
Comment (SarahAlidoost) [inline CHANGELOG.rst:39]: can we add util functions to the list here?

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@sverhoeven looks good, just some minor fixes.

File: CHANGELOG.rst (hunk 4 above)
Comment (sverhoeven) [reply CHANGELOG.rst:39]: functions in utils.py are mainly for internal use while CHANGELOG is for people use public interface of package. So I don't feel it is needed to add those.

However I could mention the extra non-pymt non-bmi methods for models.

File: CHANGELOG.rst (hunk 1 above)
Comment (sverhoeven) [reply CHANGELOG.rst:21]: added newlines to fix

**Review by SarahAlidoost — APPROVED** [review]
thanks for addressing the comments.

**Pull request merged**